docs/semgrep-assistant/privacy.md

```diff
@@ -12,17 +12,16 @@ tags:
 
 Semgrep Assistant uses API permissions to access code on your pre-selected GitHub or GitLab repositories.
 
-* Semgrep Assistant logs and stores AI (including GPT) prompts and responses for the sake of performance evaluation, which includes source code snippets.
-* Semgrep Assistant sends relevant lines of code to OpenAI's API. Currently, "relevant lines of code" means lines that are part of the Semgrep finding, plus the minimum number of lines of code required to provide enough context to produce accurate results. Semgrep, Inc. is likely to expand this, potentially to the entire file, as we learn how to pass more useful context.
+* Semgrep Assistant logs and stores AI prompts and responses for the sake of performance evaluation, which includes source code snippets.
+* Semgrep Assistant sends relevant lines of code to our AI subprocessor. Currently, "relevant lines of code" means lines that are part of the Semgrep finding, plus the minimum number of lines of code required to provide enough context to produce accurate results. Semgrep, Inc. is likely to expand this, potentially to the entire file, as we learn how to pass more useful context.
 * Semgrep stores and retains AI responses based on these code snippets for up to 6 months. Semgrep, Inc. will update you with at least a 30-day notice if we make any changes to the retention policy.<!-- markdown-link-check-disable -->
-* Semgrep, Inc. is a paying customer of OpenAI and has a Data Protection Agreement signed with them (provided upon request by [contacting support](/docs/support)). The code snippets we upload are persisted by OpenAI temporarily, following their data usage policies at [Enterprise privacy at OpenAI](https://openai.com/enterprise-privacy).<!-- markdown-link-check-enable -->
+* Semgrep, Inc. is a paying customer of its AI subprocessors and has Data Protection Agreements signed with them (these can be provided upon request by [contacting support](/docs/support)).<!-- markdown-link-check-enable -->
```

Comment:
I removed several OpenAI-specific lines (here, line 25) to make things generic. However, is this information we want to include at the bottom under subprocessor-specific headings?

Reply:
This is just my opinion so @jsmoxon may have a different take but the OpenAI-specific stuff should probably still be mentioned under some new section that goes over our specific agreements with these vendors. In the earlier PR where I removed most of the GPT mentions, I didn't remove the "we have an agreement with OpenAI" because it is, well, specific to OpenAI.